### PR TITLE
Cytosnake's `cp_process_singlecells` benchmarks with NF1 dataset

### PR DESCRIPTION
**Pull request by axiomcura**
This PR adds the benchmarks of CytoSnake's `cp_process_singlecells` workflow with the NF1 dataset.

### CODE REVIEW

File: all-benchmarks/nf1_sc_cp-process-singlecells_benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Cell-health CellProfiler features with `cp_process`
+
+## Introduction
+
+Here, we benchmark the `cp_process_singlecells` workflow with the NF1 dataset [dataset](https://github.com/WayScience/nf1_cellpainting_data) that is demonstrated in the jupyter notebook.
+
+## About the data
+
+The NF1 data repository, available at [https://github.com/WayScience/nf1_cellpainting_data](https://github.com/WayScience/nf1_cellpainting_data), houses image-based profiles assayed through CellPainting. This dataset comprises information from four plates, each annotated with metadata corresponding to specific genotypes. Genotypes are categorized as wild type (+/+) and Null (-/-).
```
Comment (MikeLippincott) [inline all-benchmarks/nf1_sc_cp-process-singlecells_benchmarks/README.md:9]: Might be worth expanding on what format the image-based profile is. Both in terms of file type and possibly structure. Might be out of scope.

File: all-benchmarks/nf1_sc_cp-process-singlecells_benchmarks/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Cell-health CellProfiler features with `cp_process`
```
Comment (MikeLippincott) [inline all-benchmarks/nf1_sc_cp-process-singlecells_benchmarks/README.md:1]:
```suggestion
# NF1 CellProfiler features with `cp_process_singlecells`
```

**Review by MikeLippincott — APPROVED** [review]
LGTM. Please see comments and suggestions prior to merging!

File: all-benchmarks/nf1_sc_cp-process-singlecells_benchmarks/README.md (hunk 1 above)
Comment (axiomcura) [reply all-benchmarks/nf1_sc_cp-process-singlecells_benchmarks/README.md:9]: Good point, but this might be out of the scope.

**Pull request merged**